content/general/concepts/api/api.md

```diff
@@ -27,3 +27,25 @@ APIs can be divided into three groups:
 - Private APIs, also known as Internal APIs, are used primarily within a company to share resources and facilitate the business. (e.g. Company Warehouse API for managing inventory with code).
 
 - Partner APIs require rights or specific licenses for use. These APIs are popular in software-as-a-service platforms (e.g. [AWS API](https://docs.aws.amazon.com/general/latest/gr/aws-apis.html)).
+
+## Benefits and Risks of Using APIs
+
+The main benefits of using APIs are:
+
+- Cost saving through the elimination of the need to build and maintain local physical infrastructure and services.
+
+- Opportunity to leverage the expertise of different vendors instead of having to reinvent from scratch things like authentication, payment processing, maps, etc.
+
+- Possibility to build new businesses based on the exchange of data between users and various online services offering API access to independent developers.
+
+The main risks associated with using APIs are:
+
+- Having no control over the vendor's infrastrucure or how the vendor allows developers to access it through the API.
+
+- Unpredictability of the vendor's internal decision-making may have a very adverse effect on businesses relying on the API.
+
+- Having little or no control over exactly what data the API collects and how it ends up being used by the API's provider.
```

Comment:
```suggestion
- Updates to an API may cause breaking changes for a product that uses it.
```

I feel like the first three list items all relate to how using an outside vendor's API leaves one at the mercy of any breaking changes that come with updates.

Reply:
I am not sure about this one. Updates are one thing, but the problem is that many more things may happen, including the vendor deciding to change their business model entirely or to misuse the data that is collected through the API.

Reply:
Gotcha. I guess when I was reading through these three points, the common things that stuck out to me was these all have to do with what the vendor does with the API ("Having little or no control...", "Unpredictability...", etc.). 

Instead, what do we think of this:

```
- Little to no control over the impact of vendor-only changes including business infrastructure, version updates, and data collection. 
```

Reply:
Yes, this looks good!

Reply:
Cool! Let's commit it.